.pre-commit-config.yaml

```diff
@@ -25,7 +25,7 @@ repos:
         name: blacken-docs
         entry: blacken-docs
         language: system
-        types_or: [python, rst, markdown]
+        types_or: [rst, markdown]
```

Comment:
ruff currently only supports formatting code examples in docstrings, not markdown or rst files.